tests/test_models.py

```diff
@@ -68,7 +71,36 @@ def test__pop_inst(client):
 
 
 def test__build_uuid_list(client):
+    """Test _build_uuid_list function."""
     rec_obj = {'items': [{'uuid': '1234'}]}
     children = 'items'
     child_list = client._build_uuid_list(rec_obj, children)
     assert '1234' in child_list
+
+
+def test_build_file_list_local():
```

Comment:
If you remove the function that this is testing this test would obviously no longer be needed. For future reference though, I'd suggest using one of pytest's [temp dir fixtures](https://docs.pytest.org/en/latest/tmpdir.html). I've found a few isolated cases where they don't quite fit, but they will generally be a better option than trying to manage temp files and temp dirs on your own.

Reply:
Great, though I hope I don't have consider too many local file operations in the future